### ISSUE TITLE
配置文件参数问题

### ISSUE THREAD

**Issue by 17695996717** [PaddlePaddle/PaddleDetection#1343]
问下下面的weights是什么路径呢？

architecture: YOLOv3
use_gpu: true
max_iters: 25000
log_smooth_window: 100
log_iter: 10
save_dir: output
snapshot_iter: 10000
metric: VOC
pretrain_weights:/ppyolo.pdparams
finetune_exclude_pretrained_params: ['cls_score','bbox_pred']
weights:

**Comment by still-wait** [comment]
@17695996717 `weights ` 是训好的模型路径，`pretrain_weights`是预训练模型的路径。

**Comment by 17695996717** [comment]
> @17695996717 `weights ` 是训好的模型路径，`pretrain_weights`是预训练模型的路径。

那save_dir呢，也是训好保存模型的路径吗

**Comment by still-wait** [comment]
@17695996717 `save_dir`指的是训练时保存模型的根目录，保存模型时会在此目录中创建以yaml配置文件名前缀命名的文件夹，里面会存放模型权重。

**Comment by still-wait** [comment]
@17695996717 有基于rcnn模型的配置文件说明文档，可以参考：https://github.com/PaddlePaddle/PaddleDetection/blob/release/0.4/docs/advanced_tutorials/config_doc/RCNN_PARAMS_DOC.md

**Comment by 17695996717** [comment]
> @17695996717 有基于rcnn模型的配置文件说明文档，可以参考：https://github.com/PaddlePaddle/PaddleDetection/blob/release/0.4/docs/advanced_tutorials/config_doc/RCNN_PARAMS_DOC.md

好的谢谢
